props/saber_fett263_buttons.h

```diff
@@ -1249,6 +1404,37 @@ SaberFett263Buttons() : PropBase() {}
     color_mode_ = NONE;
   }
 
+  // Check if current style uses RgbArg for CC_COLOR_LIST
```

Comment:
and do what?
Normally when I see a function called "CheckSOMETHING" I kind of expect it to return a bool if SOMETHING is going on.  This function doesn't return a bool, so clearly it does something if the current style uses RgbArg, but what?

Reply:
renamed to ToggleCCMode() and updated comment, we shouldn't use ColorWheel if styles use RgbArg so we need to check for argument and then use CC_COLOR_LIST instead